src/toil/jobStores/abstractJobStore.py

```diff
@@ -223,6 +223,9 @@ def exists(self, jobStoreID):
     # One year should be sufficient to finish any pipeline ;-)
     publicUrlExpiration = timedelta(days=365)
 
+    def getEnv(self):
+        return ['AZURE_ACCOUNT_KEY'] #not sure what other env variables to ask for
```

Comment:
The return value must be a dictionary, since you need to support entries with fixed value that is not looked up from the environment. Most importantly, the AzureJobStore implementation of this method needs to return a value for AZURE_ACCOUNT_KEY since that value is read from a config file, not the current directory. I outlined this in my original email.